demos/LowAllocationWebServer/HttpHeaders.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Utf8;
+using LowAllocationServer;
+
+namespace System.Net.Http.Buffered
+{
+    public struct HttpHeaders : IEnumerable<KeyValuePair<Utf8String, Utf8String>>
+    {
+        private readonly ByteSpan _bytes;
+        
+        public HttpHeaders(ByteSpan bytes)
+        {
+            _bytes = bytes;            
+        }                
+
+        public Utf8String this[string headerName]
+        {
+            get
+            {
+                var headers = this.Where(h => h.Key == headerName);
+
+                return headers.Any() ? headers.First().Value : Utf8String.Empty;
+            }
+        }
+
+        public int Count => this.Count();
+
+        public IEnumerator<KeyValuePair<Utf8String, Utf8String>> GetEnumerator()
```

Comment:
I wasn't sure if that was the case. I already have that implementation. I will pull it.